### PR TITLE
StagePasses gradle steps run the build using Java 8

### PR DESCRIPTION
**Pull request by eskatos**
Gradle steps in `StagePasses` now require Java 8

Example failure fixed by this PR: https://builds.gradle.org/viewLog.html?buildId=9787032

### CODE REVIEW

File: .teamcity/Gradle_Check/configurations/StagePasses.kt [hunk 1]
```diff
@@ -81,6 +84,9 @@ class StagePasses(model: CIBuildModel, stage: Stage, prevStage: Stage?) : BaseGr
                 buildFile = "gradle/buildTagging.gradle"
                 gradleParams = "-PteamCityUsername=%teamcity.username.restbot% -PteamCityPassword=%teamcity.password.restbot% -PteamCityBuildId=%teamcity.build.id% -PgithubToken=%github.ci.oauth.token%"
                 useGradleWrapper = true
+                params {
```
Comment (bamboo) [inline .teamcity/Gradle_Check/configurations/StagePasses.kt]: :o: Is the duplication necessary?

File: .teamcity/Gradle_Check/configurations/StagePasses.kt [hunk 2]
```diff
@@ -67,6 +67,9 @@ class StagePasses(model: CIBuildModel, stage: Stage, prevStage: Stage?) : BaseGr
             tasks = "createBuildReceipt"
             gradleParams = "-PtimestampedVersion -Djava7Home=%linux.jdk.for.gradle.compile% -Djava7.home=%linux.jdk.for.gradle.compile% --daemon"
             useGradleWrapper = true
+            params {
+                param("env.JAVA_HOME", "%linux.java8.oracle.64bit%")
```
Comment (wolfs) [inline .teamcity/Gradle_Check/configurations/StagePasses.kt]: Why don't you set top level params as here?
https://github.com/gradle/gradle/blob/4207cfec99f53cc7ff0ad8e1ee38f644cf149a5c/.teamcity/Gradle_Check/configurations/FunctionalTest.kt#L21-L27

File: .teamcity/Gradle_Check/configurations/StagePasses.kt (hunk 1 above)
Comment (eskatos) [reply .teamcity/Gradle_Check/configurations/StagePasses.kt]: Fixed

File: .teamcity/Gradle_Check/configurations/StagePasses.kt (hunk 2 above)
Comment (eskatos) [reply .teamcity/Gradle_Check/configurations/StagePasses.kt]: Fixed

**Pull request merged**